### PR TITLE
chore: refactor helm chart

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**

### CODE REVIEW

File: helm/templates/deployment.yaml [hunk 1]
```diff
@@ -91,15 +102,15 @@ spec:
                   name: {{ $s3SecretName }}
                   key: secretAccessKey
             {{- end }}
-            {{- if eq (upper .Values.rasterCommon.storage.mapproxyConfigProvider) "FS" }}
+            {{- if eq (upper $storage.mapproxyConfigProvider) "FS" }}
             - name: MAPPROXY_YAML_FILE_PATH  
               value: /configSource/mapproxy.yaml
             {{- end }} 
             - name: INTERNAL_MOUNT_DIR
```
Comment (asafMasa) [inline helm/templates/deployment.yaml:109]: isn't INTERNAL_MOUNT_DIR and ROOT_TILES_PATH uses for mapproxy config location? if its for FS why isn't it inside the If? - please check the need for them

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: helm/templates/_resources.tpl [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+{{- define "route.fullname" -}}
+{{- printf "%s-%s-%s" .Release.Name .Chart.Name "route" | indent 1 }}
+{{- end }}
+
+{{/*
+Create ingress name as used by the service name label.
+*/}}
+{{- define "ingress.fullname" -}}
+{{- printf "%s-%s-%s" .Release.Name .Chart.Name "ingress" | indent 1 }}
+{{- end }}
```
[the remark this answers is not in the document]
Comment (CL-SHLOMIKONCHA) [reply helm/templates/_resources.tpl:34]: fixed

File: helm/templates/deployment.yaml (hunk 1 above)
Comment (CL-SHLOMIKONCHA) [reply helm/templates/deployment.yaml:109]: bug from the past, its not on the scope of the PR but fixed

File: helm/templates/_helpers.tpl [hunk 3]
```diff
@@ -27,10 +27,17 @@ Create chart name and version as used by the chart label.
 {{- end }}
 
 {{/*
-Create service name as used by the service name label.
+Create configmap name as used by the service name label.
```
Comment (asafMasa) [inline helm/templates/_helpers.tpl:30]: duplicated

File: helm/templates/_helpers.tpl [hunk 4]
```diff
@@ -27,10 +27,17 @@ Create chart name and version as used by the chart label.
...
 */}}
-{{- define "service.fullname" -}}
-{{- printf "%s-%s" .Release.Name "service" }}
+{{- define "configmap.fullname" -}}
+{{- printf "%s-%s-%s" .Release.Name .Chart.Name "configmap" | indent 1 }}
+{{- end }}
+
+{{/*
+Create configmap name as used by the service name label.
```
Comment (asafMasa) [inline helm/templates/_helpers.tpl:37]: duplicated

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: helm/templates/_helpers.tpl (hunk 3 above)
Comment (CL-SHLOMIKONCHA) [reply helm/templates/_helpers.tpl:30]: fixed

File: helm/templates/_helpers.tpl (hunk 4 above)
Comment (CL-SHLOMIKONCHA) [reply helm/templates/_helpers.tpl:37]: fixed

**Review by asafMasa — APPROVED** [review]

**Pull request merged**